CHANGES.md

```diff
@@ -1,2 +1,11 @@
+Version 2.1:
+ - Change default Docker version to 1.11.2, was 1.5.0
```

Comment:
What are the chances of blowing away the default version altogether?  Would that make future upgrades less of an all-at-once thing?